### ISSUE TITLE
All incoming messages goes to SPAM folder

### ISSUE THREAD

**Issue by adrifi90** [roundcube/roundcubemail#9066]
HI,

all messages received on my e-mail box, especially those from "good" senders, end up in the spam folder.

Reading the header of the email received, you can see the following:

X-Spam-Status: Yes, score=10.7
X-Spam-Score: 107
X-Spam-Bar: ++++++++++
X-Spam-Report: Spam detection software, running on the system "nl1-sr9.supercp.com",
    has identified this incoming email as possible spam. The original
    message has been attached to this so you can view it or label
    similar future email. If you have any questions, see
    root\@localhost for details.
    Content preview: 
    Content analysis details: (10.7 points, 3.0 required)
    pts rule name description

How can I fix this problem?

**Comment by alecpl** [comment]
If you have problems with your email account (e.g. cannot log in, emails got lost, etc.) or if you have questions how to configure your Outlook or mobile phone to get email, this isn't the right place to ask. Roundcube is not a service but free software which somebody installed for you.

Please contact your internet hosting provider or IT responsible instead. If you don't know who this might be, please review your bills and find out who you're paying for email and webhosting services.

**Comment by adrifi90** [comment]
This is not a question about login or lost mails, this is a question about your software throwing all the mails I receive into spam

**Comment by alecpl** [comment]
No, Roundcube does not move mail to folders by itself. Usually it's mail server responsibility, or you're using some external plugin.
